pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[project]
+name = "xsar"
+requires-python = ">= 3.10"
+license = { text = "MIT" }
+description = "Python xarray library to use Level-1 GRD Synthetic Aperture Radar products"
+readme = "README.md"
+dependencies = [
+  'GDAL',
+  'dask[array]',
+  'distributed',
+  "xarray>=2024.10.0",
+  'affine',
+  'rasterio',
+  'cartopy',
+  'fiona',
+  'pyproj',
+  'numpy',
+  'scipy',
+  'shapely',
+  'geopandas',
+  'fsspec',
+  'aiohttp',
+  'pytz',
+  'psutil',
+  'jinja2',
+  'rioxarray',
+  'lxml',
+]
```

Comment:
In setup.py, there was an extra_requires to add other dependencies if necessary. I don't see the same thing here. Maybe not necessary. I wonder if it would not be better to always include the three SAR readers by default in the deps.

Reply:
fixed with https://github.com/umr-lops/xsar/pull/235/commits/6f834fbc87fef7747e5b778f6251f4ec1e55ab6d